libraries/AP_Notify/AP_Notify.h

```diff
@@ -248,6 +252,8 @@ class AP_Notify
     uint32_t _send_text_updated_millis; // last time text changed
     char _flight_mode_str[5];
 
+    uint8_t _send_text_src_override = 0; //Bitmask of what lines send_text_src should override
```

Comment:
I think this should be in the display backend.

Reply:
Bump.

Reply:
send_text and send_text_scripting currently treats all screens as duplicates, basically state information is common between backends, duplicating the variables in backend doesn't make sense with current structure. We are keeping standard send_text related variable here as well. So, again they all are sitting together as they should.